src/integrations/rapier/testbed_plugin.rs

```diff
@@ -88,49 +87,80 @@ impl FluidsTestbedPlugin {
         self.render_boundary_particles = enabled;
 
         for sn in self.boundary2sn.values_mut() {
-            sn.scene_node_mut().set_visible(enabled);
+            // sn.scene_node_mut().set_visible(enabled);
         }
     }
 }
 
 impl TestbedPlugin for FluidsTestbedPlugin {
-    fn init_graphics(&mut self, window: &mut Window, gen_color: &mut dyn FnMut() -> Point3<f32>) {
-        let particle_radius = self.fluids_pipeline.liquid_world.particle_radius();
+    fn init_graphics(
+        &mut self,
+        graphics: &mut GraphicsManager,
+        commands: &mut Commands,
+        meshes: &mut Assets<Mesh>,
+        materials: &mut Assets<StandardMaterial>,
+        components: &mut Query<(&mut Transform,)>,
+        harness: &mut Harness,
+        gen_color: &mut dyn FnMut() -> Point3<f32>,
+    ) {
+        let bodies = &mut harness.physics.bodies;
+        let colliders = &mut harness.physics.colliders;
 
         for (handle, fluid) in self.fluids_pipeline.liquid_world.fluids().iter() {
             let color = *self.f2color.entry(handle).or_insert_with(|| gen_color());
-            let node = FluidNode::new(particle_radius, &fluid.positions, color, window);
-            let _ = self.f2sn.insert(handle, node);
+            let _ = self
+                .f2sn
+                .insert(handle, Vec::with_capacity(fluid.positions.len()));
+
+            if let Some(entities) = self.f2sn.get_mut(&handle) {
+                for particle in &fluid.positions {
+                    //FIXME: What is the best way forward here?
```

Comment:
This is an interesting approach. However, this adds data to the body and collider sets from Rapier so it may falsify some of the performance measurements. I would suggest doing this instead:
1. In the Rapier testbed, change `EntityWithGraphics::collider` so that it becomes an `Option<ColliderHandle>`.
2. In this loop, per particle, build one `EntityWithGrahics` with its collider set to `None`.

Reply:
I will leave the RigidBody being created...

Reply:
Done